pkg/driverbuilder/docker.go

```diff
@@ -203,9 +205,17 @@ func (bp *DockerBuildProcessor) Start(b *builder.Build) error {
 		Image: builderImage,
 	}
 
+	// check if on a sles target type, which requires docker to run with `--net=host` for builder images to work
+	// for more info, see the suse container connect README: https://github.com/SUSE/container-suseconnect
+	var netMode = container.NetworkMode("default")
+	if b.TargetType == "sles" {
```

Comment:
I would add a new builders interface (Network mode() string) and only implement in for sles builder instead.
Here we can just check
```
if vv, ok := v.(NetworkModder); ok {
  NetMode = vv.NetworkMode()
}
```
I think it is more elegant and future proof this way because of the builder knowledge will belong to the builder specific source file.

Reply:
This makes way more sense 🙂 I just removed the if and set the default network mode to `"default"`, but it can be overridden in an individual builder. Then the hostConfig just has:
```
NetworkMode: container.NetworkMode(b.BuilderImageNetworkMode),
```

Let me know if you'd like it a different way or want the functions moved around. I put the default function in `builders.go`